cclient/api/extended/get_latest_inclusion.c

```diff
@@ -54,7 +54,6 @@ retcode_t iota_client_get_latest_inclusion(iota_client_service_t const* const se
 done:
   get_node_info_res_free(&node_info);
   // no need to free it, it should free via caller.
-  inclusion_req->transactions = NULL;
```

Comment:
as the comment mentioned, the `transactions` takes from the caller, better free it by the caller itself